sdk/identity/azure-identity/src/main/java/com/azure/identity/RegionalAuthority.java

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.identity;
+
+import com.azure.core.util.ExpandableStringEnum;
+
+/**
+ * Defines currently available regional authorities, or "AutoDiscoverRegion" to auto-detect the region.
+ */
+public class RegionalAuthority extends ExpandableStringEnum<RegionalAuthority> {
+    public static final RegionalAuthority AUTO_DISCOVER_REGION = fromString("AutoDiscoverRegion");
```

Comment:
These will need Javadocs

Reply:
Can we skip them? They are sort of self-explanatory.

Reply:
added docs